src/js/shared/components/Home.jsx

```diff
@@ -23,7 +21,7 @@ export default class Home extends Component {
                 <div id="selected-text">
                     <p>Get ready for</p>
                     <h1>President {candidates[winningParty].lastname}</h1>
-                    <p>According to <a href={source.url} target="_blank">{source.source}</a>, {candidates[winningParty].firstname} {candidates[winningParty].lastname} has a {source[winningPercent]}% chance of winning, while {candidates[losingParty].firstname} {candidates[losingParty].lastname} has a {source[losingPercent]}% chance of winning.</p>
+                    <p>According to <a href={source.url} target="_blank">{source.source}</a>, {candidates[winningParty].firstname} {candidates[winningParty].lastname} has a {source[winningParty]}% chance of winning, while {candidates[losingParty].firstname} {candidates[losingParty].lastname} has a {source[losingParty]}% chance of winning.</p>
```

Comment:
Can we make the "five-thirty-eight" pretty in both the select box and the link? I added a sourceName key to the API to make this easier on your end